svelte-kit-scss/src/lib/components/auth/Auth.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { render, screen } from '@testing-library/svelte';
+import userEvent from '@testing-library/user-event';
+import Auth from './Auth.svelte';
+
+describe('Auth Component', () => {
```

Comment:
This feels like it is missing additional tests
- Does a user actually get signed in? We can mock this

Reply:
Additionally, @iansamz , what happens if the signin fails? Is it handled gracefully? We should have tests for this to assert that that is indeed the behaviour.